oracle/README.md

```diff
@@ -46,13 +46,20 @@ CREATE USER datadog IDENTIFIED BY <password>;
 -- Grant access to the datadog user.
 GRANT CONNECT TO datadog;
 GRANT SELECT ON gv_$sysmetric TO datadog;
+GRANT SELECT ON sys.dba_data_files TO datadog;
+```
+
+**Note**: If you're using Oracle 11g (or lower), there's no need to run the following line:
```

Comment:
@truthbk I'm a bit concerned about the `11g or lower`, can we add a bottom line here? What's a reasonable old version we can support?

Reply:
@truthbk up :)

Reply:
@truthbk up bis 🙇

Reply:
+1 for dropping the parenthetical unless there is a strong reason not to.